integration_tests/e2e/induction/updateInductionQuestionSet.cy.ts

```diff
@@ -136,4 +150,145 @@ context(`Change Induction question set by updating the answer to 'Hoping to work
         ),
     )
   })
+
+  it(`should update a short question set Induction into a long question set Induction given form submitted with 'Hoping to work on release' as Yes`, () => {
```

Comment:
This is the cypress test that drives the scenario we are implementing - that of updating an existing Induction that was based on the short question set (where "do you want to work" was answered as No or Not Sure), and updating "do you want to work" to Yes.

"Do you want to work" is the key question that drives the question set. When it is answered as No or Not Sure we ask the "short" question set. Where it is answered as Yes we ask the long question set.

This scenario is the updating of that one question - by changing that one question we actually have to kick off the complete Induction question set in "long question set" mode to capture the answers to questions that were not previously asked